### DISCUSSION TITLE
How to disable Nitro server routes?

### DISCUSSION THREAD

**Discussion by alex00888** [nuxt/nuxt#17508] [Questions] [answered]
I need to use my own API written on Laravel at /api, but when I fetch it Nuxt fails with error "[nuxt] [request error] Failed to parse URL". How to make Nitro stop listening /api so I could use my Laravel API?

**Comment by pi0** [comment, marked as answer]
Nitro is handling all Nuxt server routes, actually! There is no specific exception for `/api/*` paths, but if you need to proxy `/api/*` to an external server (`localhost:3000/api` => `localhost:8080/api`), I'm afraid it needs to be handled by Nuxt/Nitro server unless you use a reverse proxy that splits the traffic.

**Reply by alex00888** [reply]
Yes, I'm using Nginx as a reverse proxy, but the requests from the SSR don't go to Nginx, Nitro catch all calls to /api from inside, they don't reach Nginx :(
Can I somehow disable this behaviour? I need Nuxt to make real network requests

**Reply by pi0** [reply]
> I need Nuxt to make real network requests

For this, you have to use an explicit baseURL on the server side when calling fetch instead of using relative paths. It can be `localhost` or `127.0.0.1`. 

Using shared util might make it easier:

```js
// composables/fetch.ts

export const $apiFetch = $fetch.create({ baseURL: process.client ? '/api' : 'http://localhost:8080/api' })
```

**Reply by alex00888** [reply]
I'll try it, thanks!
